ts/Stock/Indicators/MultipleLinesComposition.ts

```diff
@@ -255,8 +331,54 @@ namespace MultipleLinesComposition {
         indicator.options = mainLineOptions;
         indicator.graph = mainLinePath;
         SMAIndicator.prototype.drawGraph.call(indicator);
+
+        // Modify options and generate area fill:
+        if (this.userOptions.fillColor && areaLinesNames.length !== 0) {
+            const secondLinePoints =
+                areaLinesNames.length === 1 ?
+                    indicator.points :
+                    (indicator as any)['graph' + areaLinesNames[1]].points;
+            const fristLinePoints = (indicator as any)['graph' + areaLinesNames[0]].points;
+
+            drawArea({
```

Comment:
Q: I'm sorry, I don't quite follow this method 😅 
Code it not really that shorter than without this, neither this method used in any other place? 🤔